### PR TITLE
Update secMiddleware.py

### PR DESCRIPTION
**Pull request by goshku**
Fix for "Session reuse detected, IPAddress logged" when using Cloudflare proxy.
Original topic - https://forums.cyberpanel.net/discussion/1608/fixing-error-message-session-reuse-detected-ipaddress-logged

### CODE REVIEW

**Comment by usmannasir** [comment]
You have not assigned any value to `ip` variable but checking if it is none https://github.com/usmannasir/cyberpanel/blob/7af2ed60a4008b28b93f3af1ceb05079c57c5d98/CyberCP/secMiddleware.py#L21

Something does not seems right.

File: CyberCP/secMiddleware.py [hunk 1]
```diff
@@ -17,7 +17,9 @@ def __call__(self, request):
         try:
             uID = request.session['userID']
             admin = Administrator.objects.get(pk=uID)
-            ipAddr = request.META.get('REMOTE_ADDR')
+            ipAddr = request.META.get('HTTP_CF_CONNECTING_IP')
+            if ip is None:
```
Comment (publicarray) [inline CyberCP/secMiddleware.py:21]:
```suggestion
            if ipAddr is None:
```

**Review by publicarray — COMMENTED** [review]
There are 3 more changes needed for proper Cloudflare support:

```python
logging.writeToFile(request.META.get('REMOTE_ADDR'))
...
ipAddr = request.META.get('REMOTE_ADDR').split(':')[:3]
...
logging.writeToFile(request.META.get('REMOTE_ADDR'))
```

**Comment by publicarray** [comment]
Why was this closed?

**Pull request closed without merging**